source/install/build_from_c.sh

```
@@ -16,6 +16,28 @@ cd ${BUILD_TMP_DIR}
 cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} -DDEEPMD_C_ROOT=${DEEPMD_C_ROOT} -DLAMMPS_VERSION=stable_22Jul2025_update2 ..
 cmake --build . -j${NPROC}
 cmake --install .
+
+CONSUMER_TMP_DIR=${BUILD_TMP_DIR}/cmake-consumer
+rm -rf ${CONSUMER_TMP_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)